README.md

```diff
@@ -1,7 +1,7 @@
 Spring Boot Starter Handlebars
 ====
 
-[![Build Status](https://travis-ci.org/allegro/handlebars-spring-boot-starter.svg?branch=master)](https://travis-ci.org/allegro/handlebars-spring-boot-starter)
```

Comment:
Could you please revert it to `https://travis-ci.org/allegro/...`?

Reply:
reverted